requirements.txt

```diff
@@ -12,4 +12,6 @@ regex==2019.8.19
 sacremoses==0.0.13
 scikit_learn==0.23.2
 spacy==2.3.4
-stop_words==2018.7.23
\ No newline at end of file
+stop_words==2018.7.23
+pandas==1.1
+dask[complete]==2021.3.0
```

Comment:
Dask should be removed from requirements as we don't want to force user to install it

Reply:
agree